package.json

```diff
@@ -20,7 +20,7 @@
     "test": "jest",
     "test:watch": "jest --watch",
     "test:cov": "jest --coverage",
-    "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
+    "test:debug": "node --inspect-brk -r tsconfig-paths/register --require ts-node/register node_modules/.bin/jest --runInBand",
```

Comment:
is there a differenc between 'r' and 'require'?

Reply:
Nope, just makes it more clear. I find it's better to use full-length param names in scripts, vs short ones in shells.